Adds a Cybersun Intel Ship, the Kansatsu-Class Scout-Courier

Erikafox opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Adds a new ship to the game, the Kansatsu-Class Scout-Courier. A Cybersun ship designed for long term observation, surveying, and light cargo work. The Kansatsu has room for a crew of up to 7, but operates ideally around 5. 
The Captain is responsible for standard operations of the ship.
The Intelligence Officer is responsible for collecting and connecting data gathered by the ship, as well as being the liaison between the crew and their commanders. They also steward the special gear. Be cautious if they ask for a talk in their office.
The Field Agents are responsible for getting out and into the open, the cargo bay even comes equipped with inconspicuous black clothing to fit into the exotic environments.
<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [X] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
It's a small syndicate ship with a focus on roleplaying and player interaction. Not many things touch the spyship niche and I like to think I've nailed this in pretty well.
The Ship as viewed by a ghost
![image](https://user-images.githubusercontent.com/94164348/205151410-84625a1c-f0ab-4c9b-be6d-3088a883359f.png)

The ship in SDMM
![2022 12 22-20 24 01](https://user-images.githubusercontent.com/94164348/209270578-159959b6-8f5b-451c-ab69-af9fb3246e84.png)



<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: The Kansatsu Class Scout-Courier
add: A few cybersun outfit presets for the Scout's crew
add: A new type of safe that contains a stechkin, as well as spare magazines. For those who like to hide their tools.
config: a config file for the Kansatsu-Class Scout-Courier
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

![2022 12 22-20 24 02](https://user-images.githubusercontent.com/94164348/209270582-84468a51-91a9-4f59-8d5d-2da773615e97.png)

Erikafox committed:
I recompile the DM to see what the floor looks like and save my progress.

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
I put a bunch of words in funny places so that the crew can have the cybersun drip

Erikafox committed:
Update scoutcourier.json

Erikafox committed:
I experience a nuclear meltdown trying to find where the error in my code was

Erikafox committed:
I notice that I wrote something wrong right as I pressed commit

Erikafox committed:
capitalization slowly drives me towards ferality

Erikafox committed:
Update captain.dm

Erikafox committed:
In which I begin to see things when I close my eyes attempting to debug outfit load failures

Erikafox committed:
In which the lord savior jesus christ blesses me with a vision. I have forgotten to include a clothing/

Erikafox committed:
JUST FUCKING WORK I'M RUNNING OUT OF FUNNY COMMIT NAMES

Erikafox committed:
in which I heavily consider taking a hammer to my forehead

Erikafox committed:
its not. It's not shutting down. GORDON GET OUT OF THERE

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
radical.

Erikafox committed:
guh?????

Erikafox committed:
applying hammer to forehead

Erikafox committed:
buttons are the agents of satan

Erikafox committed:
ow

Erikafox committed:
adds a no diag for aesthetic.

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
Update scoutcourier.dmm

Erikafox committed:
holofields amiright

Erikafox committed:
please god I want to play wargame tonight

Erikafox committed:
Update head_of_personnel.dm

Erikafox committed:
we'll just fix this later

Erikafox commented:
I'll add some more namelists tomorrow, I've worked about 10 hours and need to do something else
No idea what to actually call it though.

triplezeta commented:
Naonu-class

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
wargame sucked I should've just kept working.

Erikafox committed:
Merge branch 'scout-courier' of https://github.com/Bjarl/Shiptest into scout-courier

Erikafox committed:
attaches a doohicky to the airtank so it actually connects to the pump. Also removes the last stray objects that the integration test was throwing at me.

Erikafox committed:
should actually fix air tank. Also makes the engines work. Whoops.

Erikafox commented:
no idea why the linters hate me but everything (aside from the ship flying forwards) is working now.

Erikafox committed:
changes some pixelshifting to hopefully make the linters happy

Erikafox committed:
replaces the varedited area name. Praise be to the thing that says ERROR

Erikafox committed:
adds the Space and Natural_Aggressive namelists to the Scout Courier

Erikafox committed:
adds engine shutters. Possibly makes the ship fly forwards.

Erikafox committed:
makes a snowflake safe for the intel officer. It contains their gun.

Erikafox committed:
now with firelocks! It also flies forwards (finally). Also Window cleaning.

Apogee-dev commented on code/modules/jobs/job_types/head_of_personnel.dm:
```diff
@@ -124,3 +124,15 @@
+	name = "Intelligence Officer (Syndicate)"
+
+	ears = /obj/item/radio/headset/syndicate/alt
+	uniform = /obj/item/clothing/under/syndicate/cybersun
```
Given Cybersun's status as a heavily-corporate faction, I'd suggest giving the intelligence officer some kind of formalwear instead of the Cybersun jumpsuit, as that's mostly meant for scientists and technicians. We don't have anything directly associated with Cybersun in that vein at the moment, so use something like the charcoal suit.

Apogee-dev commented on code/modules/jobs/job_types/shaft_miner.dm:
```diff
@@ -166,3 +166,13 @@
+	name = "Field Agent"
+
+	id = /obj/item/card/id/syndicate_command/crew_id
+	ears = /obj/item/radio/headset/syndicate/alt
```
Faction headsets are reserved for officer roles (in this case, captain and intelligence officer). Replace these with generic or department headsets.

Apogee-dev commented on code/modules/jobs/job_types/shaft_miner.dm:
```diff
@@ -166,3 +166,13 @@
+
+	id = /obj/item/card/id/syndicate_command/crew_id
+	ears = /obj/item/radio/headset/syndicate/alt
+	uniform = /obj/item/clothing/under/syndicate/cybersun
```
I'd consider just using regular tactical turtleneck for some of the basic jobs on this ship, since the cybersun jumpsuit is intended as lab / technical wear. Field agents in particular should stick to the more classic syndicate style.

Apogee-dev commented on code/modules/jobs/job_types/captain.dm:
```diff
@@ -166,3 +166,11 @@
+/datum/outfit/job/captain/syndicate/cybersun
+	name = "Captain (Syndicate)"
+
+	uniform = /obj/item/clothing/under/syndicate/cybersun
```
As with the intelligence officer, give the Captain a more formal uniform like the charcoal suit.

Erikafox replied:
I'm giving the captain an executive suit and the intel officer a charcoal suit.

Apogee-dev commented on _maps/configs/scoutcourier.json:
```diff
@@ -0,0 +1,33 @@
+	"namelists": ["GENERAL", "SPACE", "NATURAL_AGGRESSIVE"],
+	"map_name": "Kansatsu-Class Scout Courier",
+	"map_short_name": "Scout-Courier",
+	"map_path": "_maps/shuttles/shiptest/scoutcourier.dmm",
```
Map and config file names should follow the same format of $FACTION_$SHIPNAME to make them easy to find. In this case, please rename both files to "cybersun_kansatsu".

Apogee-dev commented on _maps/shuttles/shiptest/scoutcourier.dmm:
```diff
@@ -0,0 +1,2155 @@
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/machinery/door/airlock{
+	name = "Cryopods"
```
Consider adding a new area containing the cryopod room and bathroom, as right now almost the entire port side of the ship is one area. Also, rename airlock to "Cryopod Room" for consistency/autonaming purposes.

Apogee-dev commented on _maps/shuttles/shiptest/scoutcourier.dmm:
```diff
@@ -0,0 +1,2155 @@
+/turf/closed/wall/mineral/plastitanium,
+/area/ship/crew)
+"AM" = (
+/obj/machinery/door/airlock/centcom{
```
Replace all instances of airlock/centcom with airlock/grunge, as centcom airlocks have special effects related to phobias etc.

Apogee-dev commented on _maps/shuttles/shiptest/scoutcourier.dmm:
```diff
@@ -0,0 +1,2155 @@
+	icon_state = "1-2"
+	},
+/obj/machinery/door/airlock/centcom{
+	name = "Officer's Office"
```
Airlock name should match area name. Since there isn't an area specifically for this, you might want to use the vacant office area and just change the airlock name to "Office".

Apogee-dev commented on _maps/shuttles/shiptest/scoutcourier.dmm:
```diff
@@ -0,0 +1,2155 @@
+/turf/open/floor/plasteel/dark,
+/area/ship/crew)
+"dA" = (
+/obj/machinery/door/airlock/command{
```
Decal work in some areas of this ship could use some work. I'd recommend adding decals in front of certain doors, particularly the bridge and engineering. Additionally, replace the floors underneath the airlocks with tech floors instead of white tiles. The slight change in texture really sells the transition from room to room. Consider changing the floors underneath the bridge consoles to tech floors or telecoms floors as well to add some visual interest.

Erikafox replied:
put some more detail into engi and the command hallway.
![StrongDMM_Sxd75jtpRl](https://user-images.githubusercontent.com/94164348/185770656-fba7c7a2-b05d-4c57-beb4-d0213b9167a4.png)
 
![StrongDMM_BxscPmSlUt](https://user-images.githubusercontent.com/94164348/185770659-073be428-ca34-4a48-96ad-a95ba6dd5ff4.png)

Apogee-dev commented on _maps/shuttles/shiptest/scoutcourier.dmm:
```diff
@@ -0,0 +1,2155 @@
+	dir = 8
+	},
+/obj/effect/turf_decal/corner/opaque/syndiered/bordercee,
+/obj/machinery/suit_storage_unit/inherit,
```
Suit storage units can be given a req_access variable directly, so the reinforced glass enclosures around these aren't strictly necessary if you want to protect them. You also may want to move these SSUs into the cargo bay to avoid creating a bottleneck in the middle of the hallway.

Erikafox replied:
I'm fond of the little enclosures they're in. It really makes it feel like the thing is crammed full of gear.
![StrongDMM_t3BWMuiIv6](https://user-images.githubusercontent.com/94164348/185770610-77f5e171-9f50-4594-8ced-401e884f87dd.png)
 Is the only other spot I could think to put them without screwing up the very nice flow of the cargo bay.

Erikafox committed:
changes stuff to comply with reviews (clothing choice, file name)

Erikafox committed:
few more changes

Erikafox committed:
I keep thinking it's the kant-satsu

Erikafox committed:
decals it up a bit. Adds a staircase. Removes errant firelocks. Makes the bridge more interesting.

Erikafox committed:
forgot to un-coverall the assistant

Erikafox committed:
adds a suit/ I forgot. Thank you very much linters.

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
the DME recompiles and also I fix another spelling thing. The ship will now load.

Erikafox committed:
god I am so fucking good at this.

Erikafox committed:
In which god strikes me down for my hubris

Erikafox committed:
uncoveralls the shaft miner.

Erikafox commented:
I think I've unfucked just about everything (aside from the decals if there should be more, and the SSUs if they must be moved).
Screenshots in the main post are updated to reflect the current state of the ship.

Erikafox committed:
replaces the weird airlock contraption with dual port vents. Keeps 1 scrubber. Adds an airlock helper for the aac

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

BBBBAface commented:
What does the gameplay for the niche "intel ship" look like?

Sylphily commented:
ngl looks like another pill ship

Yellow-Mushroom commented:
I believe the combination of the sleeping room and the main meeting room will lead to the room being entirely unused due to any conversation getting drowned out by: snore*26

Latentish commented:
![image](https://user-images.githubusercontent.com/55075747/188056116-db92281a-9785-44c9-8b7c-ca553305bf49.png)
Yeah.

Latentish commented:
1. Like Yellow said, the bedroom/meetingroom combo is likely to be problematic. Please separate them.
2. I think you could still do with a bit more decalwork in those rooms that don't have them. Like the bridge for example..
3. I'd like to ask what this ship is explicitly supposed to do. What's it's niche? BBB asked this, but moreso, what makes this different than a generalist ship?

Erikafox commented:
> I believe the combination of the sleeping room and the main meeting room will lead to the room being entirely unused due to any conversation getting drowned out by: snore*26

I'll put a bed in the cryopod room and convert it into more of a rec space. Probably just expand the table a tile.

Erikafox commented:
>     2. I think you could still do with a bit more decalwork in those rooms that don't have them. Like the bridge for example..
I'll touch up the bridge some more then. I've gotten a bit more practice in since I opened this.
>     3. I'd like to ask what this ship is explicitly supposed to do. What's it's niche? BBB asked this, but moreso, what makes this different than a generalist ship?
Well it's a scout. Part of getting intelligence is roleplaying so it has the potential to not work out, but on the other hand, the gear that it has (which I may supplement) is supposed to allow people to get off and try to learn more about other groups in the area. It's fast and I like to imagine, semi stealthy. 
Ideally I'd like the flow to be - ship sees something interesting/hears something interesting -> captain and intel officer discuss it -> fax machine is used for orders if they're unsure if it warrants further investigation -> if possible, deploy field agent and schedule a recovery -> go over gathered information -> fax report in.
I've tried to prevent generalist syndrome by not giving it the gear to really do much outside of that. It has a few picks but basically no weaponry, and fairly cramped quarters.

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Adds Splines to the bridge to give it more character. The Intel Officer now gets 1x spyglass kit. Deletes 3 beds and puts 1 in the cryopod room. Little more enthusiasm to engineering.

Erikafox committed:
fixes ship config

Latentish commented:
> Well it's a scout. Part of getting intelligence is roleplaying so it has the potential to not work out, but on the other hand, the gear that it has (which I may supplement) is supposed to allow people to get off and try to learn more about other groups in the area. It's fast and I like to imagine, semi stealthy. Ideally I'd like the flow to be - ship sees something interesting/hears something interesting -> captain and intel officer discuss it -> fax machine is used for orders if they're unsure if it warrants further investigation -> if possible, deploy field agent and schedule a recovery -> go over gathered information -> fax report in. I've tried to prevent generalist syndrome by not giving it the gear to really do much outside of that. It has a few picks but basically no weaponry, and fairly cramped quarters.

So here's my issue with this, it doesn't have enough unique gear to constitute it as what you want it to be. But that's because that gear literally does not exist in game. I like this ship, I like it's layout, but as is it's just an underequipped generalist with some spying equipment. I heavily doubt players will play it the way you think they will.
That being said, we could use with some more of these items. If you can get a single extra item or machine or anything really to support this ship's niche added with this pr, I'll be happy to see that it gets merged somewhere down the line. As is I'm currently willing to give you a TM for testing changes. But I still want to see something else added here to support your niche. But this also requires coding.

Latentish commented on code/modules/jobs/job_types/assistant.dm:
```diff
@@ -195,3 +195,11 @@ Assistant
 	head = /obj/item/clothing/head/cowboy/sec/roumain/shadow
+
+/datum/outfit/job/assistant/syndicate/cyberagent
+	name = "Junior Agent (Cybersun)"
```
Please leave (parentheses) out of your job names.
This could be changed to "Junior Cybersun Agent" or something similar.

Latentish replied:
Imagine being me and being stupid.

Latentish commented on code/modules/jobs/job_types/captain.dm:
```diff
@@ -166,3 +166,11 @@
 	backpack_contents = list(/obj/item/gun/ballistic/automatic/pistol/commander=1, /obj/item/clothing/accessory/medal/gold/captain=1, /obj/item/stack/spacecash/c10000=1)
 
+/datum/outfit/job/captain/syndicate/cybersun
+	name = "Captain (Syndicate)"
```
Change this one please. "Cybersun Captain", "Cybersun Commander", etc.

Latentish replied:
Why was I born with brain'nt disease.

Latentish commented on code/modules/jobs/job_types/head_of_personnel.dm:
```diff
@@ -124,3 +124,15 @@
 	glasses = /obj/item/clothing/glasses/hud/health
+
+/datum/outfit/job/head_of_personnel/syndicate/intel
+	name = "Intelligence Officer (Syndicate)"
```
Remove parentheticals.

Latentish replied:
I'm the dumbass.

Latentish commented on code/modules/jobs/job_types/station_engineer.dm:
```diff
@@ -207,3 +207,11 @@
 	glasses = /obj/item/clothing/glasses/welding
 
+/datum/outfit/job/engineer/syndicate/cybersun
+	name = "Engineer (Cybersun)"
```
Remove Parentheticals.

Latentish replied:
I'm the fucking stupid.

Latentish commented on _maps/configs/cybersun_kansatsu.json:
```diff
@@ -0,0 +1,33 @@
+{
```
It's good practice to add the schema link here.

Latentish commented on code/game/objects/items/storage/secure.dm:
```diff
@@ -187,3 +187,12 @@
+
+/obj/item/storage/secure/safe/intel/PopulateContents()
+		. = ..()
+		new /obj/item/gun/ballistic/automatic/pistol/suppressed(src)
```
Why does the sensitive data safe have ballistics inside. Replace these with syndicate "Secret Documents" or something similar.

Latentish replied:
In addition, if you make this a real syndicate intel safe, it can be used elsewhere by others, which is good.

Erikafox replied:
It's so the intel officer's gun is hidden. They've got the only weapon on the ship and I didn't come up with any other good way to hide it. I suppose I can make a subtype of the subtype.

Latentish replied:
Don't make a subtype of this for that.. What I'd recommend is using a flatsatchel and putting it under a specially marked floorboard, making a funny secret item stash toilet, or just given them the gun since it's literally the only weapon. And still swap out of the stuff in the safe.

Latentish reviewed (changes requested):
Please add an image showing your areas.

Erikafox commented:
> But I still want to see something else added here to support your niche. But this also requires coding.

I don't have the first idea about coding, especially in byond. I'm sure I could pick it up but I'd need a fair bit of help and someone to listen to my ideasguying about things like radio bugs, linked long range communicators, and tasers hidden inside inconspicuous objects.

Ms-Mee commented:
Taser is stored in the balls

Erikafox committed:
makes engine shutters closed roundstart, renames captain

Latentish commented:
> > But I still want to see something else added here to support your niche. But this also requires coding.
> 
> I don't have the first idea about coding, especially in byond. I'm sure I could pick it up but I'd need a fair bit of help and someone to listen to my ideasguying about things like radio bugs, linked long range communicators, and tasers hidden inside inconspicuous objects.

I'm not the greatest coder, but I've gotten some code intensive projects done. The way you do it is by asking for help in coderbus or asking any of the more experienced coders for help.

Erikafox committed:
good practice in config

Erikafox committed:
Merge branch 'master' into scout-courier

Signed-off-by: Bjarl <94164348+Bjarl@users.noreply.github.com>

Erikafox committed:
makes the safe have documents + be properly subtyped.

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
Merge branch 'master' into scout-courier

Latentish commented:
As we're likely to move towards somewhat regular tests again, I'll give you a test if you get this passing checks.

Erikafox committed:
no idea why the trailing newline did that

Erikafox committed:
Merge branch 'master' into scout-courier

Signed-off-by: Bjarl <94164348+Bjarl@users.noreply.github.com>

Erikafox commented:
I fixed the linters, the new error I'm fairly sure isn't my fault.
![firefox_GbHjGJWdEi](https://user-images.githubusercontent.com/94164348/198862261-6f64298b-f3d6-494b-bba9-41fbf3330e6c.png)

Latentish commented:
> I fixed the linters, the new error I'm fairly sure isn't my fault. ![firefox_GbHjGJWdEi](https://user-images.githubusercontent.com/94164348/198862261-6f64298b-f3d6-494b-bba9-41fbf3330e6c.png)

It isn't your fault correct. but those do need to be changed. This was done because a clothing change happened between you posting this and now. Fix that so we can tm please.

Erikafox committed:
Merge branch 'master' into scout-courier

Valorium-RB commented:
It may not be a bad idea to give this ship some kind of stealth device, i.e. a chameleon projector or stealth implant. Could be neat to facilitate the stealthy, info-gathering stuff.

Erikafox commented:
I suppose it does shove it further into the niche. Thanks for the idea.

Erikafox committed:
Kantsatsu testmerge 2022??

quality control, more wallmounts, other things.

Erikafox committed:
Update cybersun_kansatsu.dmm

Erikafox commented:
Alright, they got a bit more gear, mainly 2 cham projectors, 2 stealth implants, 2 cyanide pills, an assortment of tablets for long range communications, a potted plant, a stethoscope and marking for the intel officer's safe, an cham mask, and fire alarms. I also fixed a few problems that I didn't notice. Like windows with space under them.

If I could get that test merge before this pr is 4 months old that'd be pretty cool.

Erikafox committed:
you win this one linters

Erikafox committed:
int test fix

PiperDoots commented:
Hey quick thing, you should probably change the map_short_name to Kansatsu-Class instead of scout courier to fit with the naming conventions because right now it sticks out like a sore thumb.

Erikafox committed:
Update cybersun_kansatsu.json

Daved27hundred commented:
hey uh, the first aid kits on this ship are empty. as in there is nothing in them. I do not think that is intentional.

Erikafox commented:
Incredibly not intentional. I'm already working on fixing all the random bullshit that I've seen now that it's actually being played.

ZephyrTFA commented:
Sounds like a skill issue to me

Erikafox committed:
A valuable lesson. Undergearing isn't fun for anyone but me while I'm making scenarios up in my head.

also fixes.

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox commented:
Screenies updated, main PR body is too

I added some more decals, gave the intelligence officer a *lot* more gear (Agent cards, cham masks, lots of cyanide pills) to hand out (on top of making the safe obvious via note). Added proper medical kits and gave the locker a name so it's obvious, and swapped the glass tables in the crew quarters for wooden ones (so you can do emergency surgery on them). And two cham setes in the captain locked gear box.

It honestly might be a bit overgeared now, but it's all things that I felt fit in, and help the ship fit in it's niche.

Erikafox committed:
Merge branch 'master' into scout-courier

Erikafox committed:
more playtest fixes

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into scout-courier

Erikafox committed:
vent fix, posters

Erikafox committed:
the fire extinguisher update

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into scout-courier

Erikafox committed:
renames the files to fit into the new schema

Erikafox committed:
the floor safe visibility update

Erikafox committed:
these things are my bane.

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into scout-courier

Erikafox committed:
less atmos problems + smaller and easier to use generator

Erikafox committed:
they probably need some fuel

Latentish commented:
I think that if this were merged as is, it should probably be admin only. Spying on others ship, or reconnaissance ship aren't things we really have much gameplay loop or content for right now. I'll actually test it this uptime to see how it does in a live round, and basically yell at people to form a crew on it. So tm'able, like I shoulda done a while ago.

Erikafox committed:
it starts with power now

Even less ass to play

Erikafox committed:
admeme spawn

Latentish commented:
mergma? What's mergma? mergma balls
